HHS/Head-Start-TTADP
# [TTAHUB-2147] Push userData to GA

## Description of change

Custom React hook to push user dimensions to GA & GTM.

![screenshot of dev console showing data layer is modified](https://github.com/HHS/Head-Start-TTADP/assets/3288586/cc257a46-c19e-4b59-ac1e-4444f6047696)

![screenshot of google tag assistant showing the data layer](https://github.com/HHS/Head-Start-TTADP/assets/3288586/f33ecc4d-6742-4a7c-9280-bf2bdf7abcaf)

## How to test

- Use [Google Tag Assistant](https://tagassistant.google.com/) to view the data layer on dev

Note that the ticket requested the data layer be updated after the container loads. Given our architecture this is, I believe, impossible. (We fetch the user data asynchronously). 

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2147


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
